src/mono/nuget/Microsoft.NET.Sdk.Mono.Toolchain.Manifest/WorkloadManifest.json.in

```diff
@@ -0,0 +1,201 @@
+{
+  "version": 1,
+  "workloads": {
+    "microsoft-net-runtime-mono-browser-build": {
+      "description": "Browser build workload",
+      "packs": [
+        "Microsoft.NET.Runtime.WebAssembly.Sdk",
+        "Microsoft.NETCore.App.Runtime.Aot.Cross.browser-wasm",
+        "Microsoft.NET.Runtime.Emscripten.Node",
+        "Microsoft.NET.Runtime.Emscripten.Python",
+        "Microsoft.NET.Runtime.Emscripten.Sdk"
+      ],
+    },
+    "microsoft-net-runtime-mono-browser-aot": {
+      "description": "Browser AoT Workload",
+      "extends": [
+        "microsoft-net-runtime-mono-browser-build",
+        "microsoft-net-runtime-mono-aot-build"
+      ]
+    },
+    "microsoft-net-runtime-mono-aot-build": {
+      "description": "Mono AOT Build",
+      "packs": [
+        "Microsoft.NET.Runtime.MonoAOTCompiler.Task",
+      ],
+    },
+    "microsoft-net-runtime-cross-android": {
+      "packs": [
+        "Microsoft.NETCore.App.Runtime.AOT.osx-x64.Cross.android-x86",
+        "Microsoft.NETCore.App.Runtime.AOT.Cross.android-x64",
+        "Microsoft.NETCore.App.Runtime.AOT.Cross.android-arm",
+        "Microsoft.NETCore.App.Runtime.AOT.Cross.android-arm64"
+      ],
+      "extends": [
+        "microsoft-net-runtime-mono-aot-build",
+        "microsoft-android-sdk-full"
+      ]
+    },
+    "microsoft-net-runtime-cross-ios": {
+      "packs": [
+        "Microsoft.NETCore.App.Runtime.AOT.Cross.ios-arm",
+        "Microsoft.NETCore.App.Runtime.AOT.Cross.ios-arm64",
+        "Microsoft.NETCore.App.Runtime.AOT.Cross.ios-x86",
```

Comment:
^ for x86 that would be a simulator build

Reply:
@spouliot is the same true for ios-x64 and tvos-x64?

Reply:
yup, same